ts/features/walletV3/payment/screens/WalletPaymentDetailScreen.tsx

```diff
@@ -105,10 +106,21 @@ const WalletPaymentDetailContent = ({
 }: WalletPaymentDetailContentProps) => {
   const navigation = useNavigation<IOStackNavigationProp<AppParamsList>>();
 
-  useHeaderSecondLevel({ title: "", goBack: undefined });
+  useLayoutEffect(() => {
+    navigation.setOptions({
+      headerShown: true
+    });
+  }, [navigation]);
+
+  useHeaderSecondLevel({
+    title: "",
+    goBack: undefined,
+    supportRequest: true,
+    contextualHelp: emptyContextualHelp
+  });
```

Comment:
```suggestion
  useHeaderSecondLevel({
    title: "",
    supportRequest: true,
    contextualHelp: emptyContextualHelp
  });
```

Reply:
Fixed in https://github.com/pagopa/io-app/pull/5276/commits/ab275b1d35c55724e368100b517a60e932123427